src/shared/utils/ownerHelpers.ts

```diff
@@ -1,4 +1,8 @@
-export function getOwnerImg(provider, owner) {
+export function getOwnerImg(provider: string, owner: string | null) {
+  if (!owner) {
```

Comment:
Looking at the other references we would rather return the default than return an incorrect github page so I felt like this was worth adding